internal/security/secrets/option/constant.go

```diff
@@ -38,7 +38,7 @@ const (
 	tlsSecretFileName             = "server.key"
 	tlsCertFileName               = "server.crt"
 	caCertFileName                = "ca.pem"
-	pkiInitFilePerServiceComplete = ".security-secrets-secrets.complete"
+	pkiInitFilePerServiceComplete = ".security-secrets-setup.complete"
```

Comment:
DOH!

Reply:
this was introduced as mistake last time when the package name was renamed from setup to secrets... so now I changed it back...